### PR TITLE
Cassandra 18006: Several CQLSH Completions are incomplete

### PR DESCRIPTION
**Pull request by bschoening**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: pylib/cqlshlib/cqlshhandling.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def registrator(f):
 '''
 
 cqlsh_capture_cmd_syntax_rules = r'''
-<captureCommand> ::= "CAPTURE" ( fname=( <stringLiteral> | "OFF" ) )?
+<captureCommand> ::= "CAPTURE" ( fname=( <stringLiteral>) | "OFF" )?
```
Comment (jeffwidman) [inline pylib/cqlshlib/cqlshhandling.py:144]: The spaces between parens are inconsistent here: `( <stringLiteral>)`

**Comment by bschoening** [comment]
Yeah, that's inconsistent.  Jira 18006 was closed as a duplicate, however.
It looks like this PR just wasn't closed.

Btw, I'd like to upgrade CQLSH from optparse to argparse, but I've tried
twice and the code is just too convoluted for me to figure it out thus far.

**Comment by jeffwidman** [comment]
So @bschoening should you close this PR then? You can do that as the author of the PR...

**Comment by bschoening** [comment]
@jeffwidman done!

**Comment by bschoening** [comment]
I've noticed that our CQL documentation has artifacts which appear to be
from an earlier markdown format and not fully converted.

I'd like to fix this, but I'm not sure what the intent was with 'bc' & 'p'


https://cassandra.apache.org/doc/stable/cassandra/cql/cql_singlefile.html#types


Examples:


"\{toc:maxLevel=3}


CQL Syntax"


bc(syntax). ::= TERMINAL


bc(syntax)..

::= CREATE KEYSPACE (IF NOT EXISTS)? WITH

p.bc(sample). SELECT sample_usage FROM cql;bc(syntax)..

::= CREATE KEYSPACE (IF NOT EXISTS)? WITH


p.

**Pull request closed without merging**